activiti-cloud-services-runtime-bundle/activiti-cloud-services-rest-api/src/main/java/org/activiti/cloud/services/rest/api/TaskController.java

```diff
@@ -37,6 +38,10 @@
     Resource<CloudTask> completeTask(@PathVariable String taskId,
                                       @RequestBody(required = false) CompleteTaskPayload completeTaskPayload);
 
+    @RequestMapping(value = "/{taskId}/save", method = RequestMethod.POST)
+    void saveTask(@PathVariable String taskId,
+                  @RequestBody(required = false) SaveTaskPayload saveTaskPayload);
```

Comment:
@igdianov any specific reason to make `SaveTaskPayLoad` optional? I the case of complete task it makes sense because we can complete a task without providing any variable. In case of `save`, it sounds mandatory to me, as calling `save` without payload will have no effect.

Reply:
@erdemedeiros Done. See https://github.com/Activiti/activiti-cloud-runtime-bundle-service/pull/296/commits/a5cb18c167610470ba083dfb97232033aec3d00c